docs/devhubOnMinishift.md

```diff
@@ -0,0 +1,91 @@
+ #Setup Docker in localhost
```

Comment:
Over all good. I think this document should not live in the Devhub though. This should be in devhub-resources.